### DISCUSSION TITLE
Fedora Asahi Linux support (Linux on Apple silicon)

### DISCUSSION THREAD

**Discussion by E-man-dev** [Heroic-Games-Launcher/HeroicGamesLauncher#4061] [Ideas]
With the recent announcement of Asahi Linux being able to run games with Proton (announcement: [https://asahilinux.org/2024/10/aaa-gaming-on-asahi-linux/](https://asahilinux.org/2024/10/aaa-gaming-on-asahi-linux/)), I was hoping Heroic Games Launcher could add support for Asahi Linux. As per the info page ([https://docs.fedoraproject.org/en-US/fedora-asahi-remix/x86-support/](https://docs.fedoraproject.org/en-US/fedora-asahi-remix/x86-support/)), it looks like Proton games still need to be run through Steam for now, but Heroic could get around this by adding the games to Steam.

**Comment by E-man-dev** [comment]
Looks like it can now run fine under muvm with the x86 AppImage, though on 2.15.2 sometimes the web pages grey out, which does not affect launching games but can make accessing store websites and login pages difficult. Would still be nice to get a native version.

**Reply by Fab-8A** [reply]
Do you have a tutorial how to install it then?

**Reply by E-man-dev** [reply]
Sorry for the delayed response! Unfortunately, it broke since I last made this post. Something something Electron defaults to compiling 4k instead of 16k pagefile size, so it does not work. You _can_ compile an ARM version, I just don't know what build flag or something you need for it to compile a 16k version.
You can still use Legendary though! Legendary is what Heroic uses for the low-level stuff for Epic Games, and Legendary can be installed from the dnf package. You will still need a Wine version that supports 16k to 4k translation... I will refer you to r/AsahiGaming for more on that.

**Comment by bauer-gotwig** [comment]
I recently tried Heroic Launcher , and no the App image is broken and does not work anymore.

**Comment by E-man-dev** [comment]
Yeah, it's pretty sad. Electron apps don't really work on Asahi since quite a few patches ago. Something about pagefile sizes. Legendary still works though, and Legendary is already part of the dnf repo for Fedora Asahi Remix. You will need a special version of Wine that has the 16k to 4k though.
